spec/javascript/containers/RestaurantFormContainerSpec.js

```diff
@@ -14,12 +14,10 @@ describe('RestaurantFormContainer', () => {
       email: "foo@foo.com",
       website: "www.foo.com"
     };
-
-
     wrapper = mount(
         <RestaurantFormContainer
           cuisine_id = {1}
-          updateParent = {console.log('updateParent')}
+          updateRestaurantsList = {console.log('updateRestaurantsList')}
```

Comment:
this is a little weird. Passing down a console log as a prop? the log would be executed immediately, and you would be passing down nil